### ISSUE TITLE
Cannot find module 'upath'

### ISSUE THREAD

**Issue by BiosSun** [johnsoncodehk/volar#204]
```
Cannot find module 'upath'
Require stack:
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vscode-typescript-languageservice/out/services/completion.js
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vscode-typescript-languageservice/out/index.js
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vscode-vue-languageservice/out/services/semanticTokens.js
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vscode-vue-languageservice/out/index.js
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vue-tsc/proxy.js
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vue-tsc/typescript/lib/tsc.js
- /Users/biossun/Develops/Kwai/radar-fe.next/node_modules/vue-tsc/typescript/bin/tsc
```

**Comment by johnsoncodehk** [comment]
Could you provide a repro case? Try this template: https://github.com/johnsoncodehk/volar-starter

**Comment by johnsoncodehk** [comment]
Fixed in 0.25.12, thanks for your report!
